djreact/djangoapps/trips/tests/test_models.py

```diff
@@ -12,8 +19,7 @@ class TestHost(TestCase):
     """
 
     def setUp(self):
-        self.host = Host(name="Arbisoft")
-        self.host.save()
+        self.host = HostFactory()
```

Comment:
Do the tests pass if we only define the factory here and don't build it?

Reply:
Check them locally. Working fine

Reply:
If you look at the factory class, it populates the fields defined inside them with the dummy values. 
Yes it is passing locally.